Update slack info for maintainers (#338)
blitz-js/blitzjs.com: docs/maintainers.mdx

- All `#-*` channels are for Blitz users
- All `#contributor-*` channels are for Blitz internal development

> - All `#lang-*` channels are for Japanese speakers (In yours country speaker channel, monitoring as possible.)

Reviewer: ```suggestion
- All `#lang-*` channels are for speakers of other languages (if you speak the language, please monitor it)
```

Author: I mistake.. thanks
